dspace-api/src/main/java/org/dspace/content/dao/impl/MetadataFieldDAOImpl.java

```diff
@@ -71,6 +71,20 @@ public MetadataField findByElement(Context context, String metadataSchema, Strin
     }
 
     @Override
+    public List<MetadataField> findByUnqualifiedElement(Context context, String metadataSchema, String element) throws SQLException
+    {
+        Criteria criteria = createCriteria(context, MetadataField.class);
+        criteria.createAlias("metadataSchema", "s").add(
+                Restrictions.and(
+                        Restrictions.eq("s.name", metadataSchema),
+                        Restrictions.eq("element", element)
```

Comment:
I will change the method name to findFieldsByElementNameUnqualified.  The method above ignores the qualifier field entirely so it may return more than one item.